File: PYTHON/2227 - Airport.py
caso = 0
while True:
    caso+=1
    a,b = map(int,input().split())
    #v = []
    vf = [0]*(a+1)
    v = []

    if b == a and a == 0:
        break
    
    for i in range(b):
        x,y = map(int,input().split())
        vf[x-1] +=1
        vf [y-1] +=1

    mai = max(vf)

    for i in range(len(vf)):
        if vf[i] == mai:
            v.append(i+1)
    print('Teste',caso)
    #v.sort()


    print(*v,sep=' ',end=' \n')
    print()
    

File: C++/2227 - Airport.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    long a,b,caso=0;
    
    while (true)
    {
        caso +=1;
        cin>>a>>b;
        if (a+b == 0)break;

        vector<int> v(a);

        for (int i =0; i<b;i++){
            long x;
            cin>>x;
            v[x-1]+=1;
            cin>>x;
            v[x-1]+=1;
        }

        long mai = 0;
        cout<<"Teste "<<caso<<endl;
        for (int i:v) if (i>mai) mai = i;

        for (int i = 0; i<a;i++){
            if (v[i] == mai) cout<<i+1<<" ";
        }
        cout<<endl;
        cout<<endl;
    }
    return 0;
    
}